### PR TITLE
docs(web-modeler): document file download and export options

### PR DESCRIPTION
**Pull request by jfriedenstab**
## Description
Documents the file download and export options in Web Modeler.

Part of https://github.com/camunda/product-hub/issues/1959.
Closes https://github.com/camunda/web-modeler/issues/5286.

## When should this change go live?

- [x] This change is not yet live and should not be merged until Feb 13 (`8.5.0-alpha1` release date) (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/components/modeler/web-modeler/file-download.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import MultiFileDownloadImg from './img/download/multi-file-download.png'
+import DownloadBpmnImg from './img/download/download-bpmn.png'
+import DownloadDmnImg from './img/download/download-dmn.png'
+import DownloadFormImg from './img/download/download-form.png'
+import DownloadConnectorTemplateImg from './img/download/download-connector-template.png'
+
+<span class="badge badge--cloud">Camunda 8 only</span>
+
+You can download all the files stored in Web Modeler to your local computer.
+For BPMN diagrams, you also have the option to export them in different image formats.
```
Comment (HanselIdes) [inline docs/components/modeler/web-modeler/file-download.md:17]:
```suggestion
You may also export BPMN diagrams in different image formats.
```
suggestion for brevity

File: docs/components/modeler/web-modeler/file-download.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+import SingleFileDownloadImg from './img/download/single-file-download.png'
+import MultiFileDownloadImg from './img/download/multi-file-download.png'
+import DownloadBpmnImg from './img/download/download-bpmn.png'
+import DownloadDmnImg from './img/download/download-dmn.png'
+import DownloadFormImg from './img/download/download-form.png'
+import DownloadConnectorTemplateImg from './img/download/download-connector-template.png'
+
+<span class="badge badge--cloud">Camunda 8 only</span>
+
+You can download all the files stored in Web Modeler to your local computer.
```
Comment (HanselIdes) [inline docs/components/modeler/web-modeler/file-download.md:16]:
```suggestion
You can download all the files stored in Web Modeler to your local computer for offline editing or integration into version control.
```
introduces the main use cases for download

**Review by HanselIdes — CHANGES_REQUESTED** [review]
Overall LGTM - I suggested some ways to clarify the value and shorten the text. Overall, the UI for this kind of feature should be self-explanatory enough that documentation can be light. The main 'gotchas' were covered nicely: the download limit and different behavior between project view and file editor view.

You do not need to re-request my review.

**Review by christinaausley — APPROVED** [review]
Nice writing here!

**Comment by jfriedenstab** [comment]
@HanselIdes To allow me to merge the PR, I think you need to approve it (because you requested changes).

**Pull request merged**